packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -90,6 +90,11 @@ exports.onPostBuild = (args, pluginOptions) => {
         urlPattern: /(\.js$|\.css$|static\/)/,
         handler: `cacheFirst`,
       },
+      {
+        // page-data.json files are not content hashed
+        urlPattern: /^https:?:*page-data\.json/,
```

Comment:
This didn't seem to work when I tested it — something like this did https://regex101.com/r/WgnMvi/1

Reply:
Ahh interesting, I thought I saw it working locally, but I'll try again. With these changes.

Reply:
Ok, fixed and actually tested changing `page-data.json` to make sure the new results are shown on the first network request